betty/assets/locale/ar/betty.po

```diff
@@ -856,6 +904,11 @@ msgstr ""
 msgid "The following errors occurred"
 msgstr ""
 
+msgid ""
+"The path to a Betty project configuration file. Defaults to "
+"betty.json|yaml|yml in the current working directory."
```

Comment:
Make the default a variable that is injected

Reply:
Ensure formats declare their extensions in order (a sequence) because that's how we scan for files.